### PR TITLE
Fix typos in 4.0 test report

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: benchmark/benchmark-sysbench-v4-vs-v3.md [hunk 1]
```diff
@@ -102,13 +102,15 @@ set global tidb_disable_txn_auto_retry=0;
 ## 测试方案
 
 1. 通过 TiUP 部署 TiDB v4.0 和 v3.0。
-2. 通过 Sysbench 导入 16 张表，每张表数据 1000 万数据。
+2. 通过 Sysbench 导入 16 张表，每张表数据 1000 行数据。
```
Comment (ran-huang) [inline benchmark/benchmark-sysbench-v4-vs-v3.md:105]: 两个`数据`?

Comment (TomShawn) [reply benchmark/benchmark-sysbench-v4-vs-v3.md:105]: @scsldb PTAL

Comment (TomShawn) [inline benchmark/benchmark-sysbench-v4-vs-v3.md:105]:
```suggestion
2. 通过 Sysbench 导入 16 张表，每张表有 1000 万行数据。
```

**Pull request merged**